### DISCUSSION TITLE
Spanish Translation

### DISCUSSION THREAD

**Discussion by avelixo** [ChristopherKlay/StadiaEnhanced#67] [Translations]
LANG: es

"default": "Por defecto",
"native": "Nativa",
"hide": "Ocultar",
"show": "Mostrar",
"visible": "Visible",
"hidden": "Oculto",
"enabled": "Activado",
"disabled": "Desactivado",
"auto": "Automático",
"manual": "Manual",
"games": "Juegos",
"bundles": "Paquetes",
"addons": "Complementos",
"responsive": "Adaptativo",
"windowed": "Modo Ventana",
"fullscreen": "Pantalla Completa",
"searchstore": "Buscar en la Tienda",
"onsale": "En Oferta",
"prodeals": "En Oferta (Pro)",
"allgames": "Todos los juegos",
"usermedia": "Capturas de pantalla y Vídeos",
"searchbtnbase": "Buscar en",
"avatarpopup": "URL del nuevo avatar (vacío por defecto):",
"searchheader": "Juegos que contienen en el título",
"sessiontime": "Duración de la sesión",
"codec": "Códec",
"resolution": "Resolución",
"hardware": "Hardware",
"software": "Software",
"trafficsession": "Tráfico de la sesión",
"trafficcurrent": "Tráfico actual",
"trafficaverage": "Tráfico promedio",
"packetloss": "Paquetes perdidos",
"framedrop": "Imágenes perdidas",
"latency": "Latencia",
"jitter": "Jitter Buffer",
"decodetime": "Tiempo de decodificación",
"compression": "Compresión",
"streammon": "Monitor de retransmisión",
"stream": "Retransmisión",
"community": "Comunidad",
"speedtest": "Test de Velocidad",
"quickaccess": "Acceso Rápido",
"messages": "Mensajes",
"avatar": "Avatar",
"interface": "Interfaz",
"shortcut": "Accesos Directos",
"shortcuttitle": "Instala un acceso directo para",
"shortcutdesc": "Permite instalar un acceso directo de un juego en tu dispositivo.",
"gridsize": "Tamaño de la cuadrícula",
"griddesc": "Cambia la cantidad de juegos por fila en la pantalla de inicio.",
"clock": "Reloj",
"clockdesc": "Muestra la hora actual en la lista de amigos, como superposición dentro del juego o ambas cosas.",
"friendslist": "Lista de amigos",
"igoverlay": "Superposición dentro del juego",
"listoverlay": "Lista y Superposición",
"filter": "Filtro",
"filterdesc": "Permite ocultar juegos de la pantalla de inicio. Activado el filtro, aparecen unos símbolos (ojos); el símbolo general, situado encima de tu colección de juegos en la parte superior derecha, aplica o desaplica los cambios en el filtro; los símbolos específicos de cada juego permiten elegir qué juegos ocultar.",
"filtertoggle": "Activado (modo normal)",
"filterquick": "Activado (modo rápido)",
"invitebase": "Copiar el enlace de invitación",
"inviteactive": "¡Copiado!",
"prolabel": "Etiqueta Pro",
"prolabeldesc": "Elimina la etiqueta 'Pro' de los juegos de la pantalla de inicio.",
"homegallery": "Galería de Capturas",
"homegallerydesc": "Oculta el área de 'Capturas' de la parte inferior de la pantalla de inicio.",
"quickprev": "Previsualización de Mensajes",
"quickprevdesc": "Oculta la previsualización de mensajes de la lista de amigos.",
"quickrep": "Respuesta Rápida",
"quickrepdesc": "Oculta la opción de respuesta rápida del chat.",
"offlinefriend": "Amigos Desconectados",
"offlinefrienddesc": "Oculta los amigos desconectados de la lista de amigos.",
"invisiblefriend": "Amigos Invisibles",
"invisiblefrienddesc": "Oculta los amigos con un estado en línea desconocido de la lista de amigos.",
"streammode": "Modo Retransmisión",
"streammodedesc": "Permite ocultar ciertos elementos (por ejemplo: la lista de amigos) mientras retransmites (a través de un programa externo como OBS o Discord).",
"catprev": "Previsualización de Categorías",
"catprevdesc": "Oculta las etiquetas de categoría cuando el cursor pasa sobre un juego de la página de inicio.",
"popup": "Efecto Zoom",
"popupdesc": "Desactiva el efecto de ampliación (zoom-in) que se produce cuando el cursor pasa sobre un juego de la página de inicio.",
"streammondesc": "Activa automáticamente el monitor de retransmisión al ejecutar un juego.",
"resolutiondesc": "Máxima resolución a la que pueden alcanzar los juegos. Para resoluciones 1440p (Quad HD) o 2160p (4K UHD) es necesario el códec VP9.",
"codecdesc": "El códec usado por los juegos.",
"confirmreset": "¿Estás seguro de querer restablecer los ajustes?",
"gamesfinished": "Juegos Completados",
"achievementsunlocked": "Logros Desbloqueados",
"splitstore": "Tienda a Doble Columna",
"splitstoredesc": "Divide la lista de la tienda en dos columnas para una mayor legibilidad.",
"scrollbar": "Barras de desplazamiento",
"scrollbardesc": "Habilita las barras de desplazamiento en la pantalla de inicio y en la tienda.",
"resetsettings": "Restablecer los ajustes"

**Comment by ChristopherKlay** [comment]
Will be including in ´1.8.2´ going into verification later today (:
Thank you!

**Reply by avelixo** [reply]
You're welcome. ;)

**Reply by avelixo** [reply]
I have improved my translation.  I'm sorry I didn't correct it before you released the patch.

**Reply by ChristopherKlay** [reply]
All fine!

The updated version will be included in the next update (:

**Reply by avelixo** [reply]
Some corrections.

**Reply by ChristopherKlay** [reply]
Will be included (:

**Comment by ChristopherKlay** [comment]
Small update of the translation file for `1.8.4` [here](https://github.com/ChristopherKlay/StadiaEnhanced/blob/master/translations.md#new-content).
No stress however; i'll update translation when available (:

**Reply by avelixo** [reply]
Done! ;)

**Reply by avelixo** [reply]
Edited.

"hardware": "Hardware",
"software": "Software",
"decodetime": "Tiempo de decodificación",

And other corrections.

**Comment by rdrenedo** [comment]
Hello, I am uploading a full version based on the previous one, with minor corrections and modifications.

LANG: es
[es.zip](https://github.com/ChristopherKlay/StadiaEnhanced/files/7796552/es.zip)

Thanks for your grat job!!!

**Reply by ChristopherKlay** [reply]
The changes will be included in the next patch.

Thank you! (:

**Reply by rdrenedo** [reply]
Hi @ChristopherKlay , I am attaching a new version with minor corrections, now it is more understandable.
Sorry to upload 2 versions in a short time...

LANG: es
[es.zip](https://github.com/ChristopherKlay/StadiaEnhanced/files/7798801/es.zip)

Happy New Year 2022!!!

**Comment by rdrenedo** [comment]
Hi!!
New Spanish update with missing translations:

    es: { // https://github.com/ChristopherKlay/StadiaEnhanced/discussions/67
        default: 'Por defecto',
        native: 'Nativa',
        hide: 'Ocultar',
        show: 'Mostrar',
        total: 'Total',
        visible: 'Visible',
        hidden: 'Oculto',
        enabled: 'Activado',
        disabled: 'Desactivado',
        auto: 'Automático',
        manual: 'Manual',
        all: 'Todo',
        locked: 'Bloqueado',
        complete: 'Completo',
        incomplete: 'Incompleto',
        games: 'Juegos',
        allgames: 'Todos los juegos',
        leavepro: 'Abandonan Pro',
        bundles: 'Paquetes',
        addons: 'Complementos',
        wishlist: 'Lista de deseos',
        responsive: 'Adaptativo',
        windowed: 'Modo ventana',
        fullscreen: 'Pantalla completa',
        onsale: 'En oferta',
        prodeals: 'En oferta Pro',
        trials: 'Versiones de prueba',
        userprofile: 'Mi perfil',
        usermedia: 'Capturas de pantalla y vídeos',
        searchbtnbase: 'Buscar en',
        avatarpopup: 'URL del nuevo avatar (vacío por defecto):',
        date: 'Fecha',
        time: 'Hora',
        sessiontime: 'Duración de la sesión',
        codec: 'Códec',
        resolution: 'Resolución',
        hardware: 'Hardware',
        software: 'Software',
        trafficsession: 'Tráfico de la sesión',
        trafficcurrent: 'Tráfico actual',
        trafficaverage: 'Tráfico promedio',
        packetloss: 'Paquetes perdidos',
        framedrop: 'Imágenes perdidas',
        latency: 'Latencia',
        jitter: 'Jitter Buffer',
        decodetime: 'Tiempo de decodificación',
        compression: 'Compresión',
        detailed: 'Detallado',
        compact: 'Compacto',	
        streammon: 'Monitor de retransmisión',
        stream: 'Retransmisión',
        network: 'Red',
        session: 'Sesión',
        extdetail: 'Detalles extendidos',
        maxresolution: 'Resolución máxima',
        fps: 'FPS',
        testdiscl: '<b>Aviso:</b> Este juego aún no se ha probado.',
        datadiscl: 'Esta es la velocidad máxima de fotogramas que se logra al jugar en modo 4K (requiere Stadia Pro). \
                    En los juegos con interruptor "resolución/rendimiento", se eligió resolución. \
                    Estos datos son proporcionados por <a href="https://twitter.com/OriginaIPenguin" target="_blank">@OriginaIPenguin</a> \
                    y se puede encontrar la base de datos completa <a href="https://linktr.ee/StadiaDatabase" target="_blank">aquí</a>.',
        noteOne: 'Modo 4K',
        noteTwo: 'Interruptor 30/60 FPS',
        noteThree: '60 FPS en modo 1080p',
        noteFour: '30 FPS en modo 1080p',
        noteFive: 'No es compatible con modo 4K',
        unsupported: 'No es compatible',
        crossfriends: 'Sin sistema Crossplay',
        filtersettings: 'Ajustes de filtro',
        saturation: 'Saturación',
        contrast: 'Contraste',
        brightness: 'Brillo',
        sharpen: 'Nitidez',
        community: 'Comunidad Stadia',
        speedtest: 'Test de velocidad',
        quickaccess: 'Acceso rápido',
        messages: 'Mensajes',
        comfeature: 'Comunidad',
        avatar: 'Avatar',
        interface: 'Interfaz',
        shortcut: 'Icono Stadia',
        shortcuttitle: 'Instala un acceso directo para',
        shortcutdesc: 'Permite instalar un acceso directo de un juego en tu dispositivo',
        stadiadatabase: 'Detalles extendidos',
        stadiadatabasedesc: 'Muestra una sección de "Detalles extendidos" en la página de la tienda de juegos, que muestra la velocidad de fotogramas, la resolución y más sobre el juego',
        contactdev: 'Contactar con el desarrollador',
        stadiahunters: 'Stadia Hunters',
        stadiahuntersdesc: 'Acceso a la comunidad de Stadia Hunters, que incluye seguimiento de logros, guías, tablas de clasificación y más. El compañero perfecto para los cazadores de logros',
        stadiahunterstitle: 'en Stadia Hunters',
        stadiahunterslogin: 'Iniciar sesión',
        stadiahuntersnotfound: 'Usuario no encontrado',
        stadiahunterslevel: 'Nivel',
        stadiahuntersworldrank: 'Rango mundial',
        stadiahuntersxphover: 'Progreso de nivel',
        gridsize: 'Tamaño de la cuadrícula',
        griddesc: 'Cambia la cantidad de juegos por fila en la biblioteca',
        clock: 'Reloj',
        clockdesc: 'Muestra la hora actual en la lista de amigos, como superposición dentro del juego o ambas cosas',
        friendslist: 'Lista de amigos',
        igoverlay: 'Superposición dentro del juego',
        listoverlay: 'Lista y superposición',
        filter: 'Filtro de juegos',
        filterdesc: 'Te permite ordenar tu biblioteca escondiendo juegos. El filtro se puede alternar a través del símbolo, arriba a la derecha, arriba de tus juegos en la biblioteca',
        invitebase: 'Copiar el enlace de invitación',
        inviteactive: '¡Copiado!',
        gamelabel: 'Etiqueta Pro',
        gamelabeldesc: 'Elimina etiquetas como "Pro" de los juegos en la pantalla de inicio',
        dimoverlay: 'Superposición atenuada',
        dimoverlaydesc: 'Elimina el efecto de atenuación al abrir el menú de Stadia durante el juego',
        homegallery: 'Galería de capturas',
        homegallerydesc: 'Oculta el área de "Capturas" de la parte inferior de la pantalla de inicio',
        quickprev: 'Previsualización de mensajes',
        quickprevdesc: 'Oculta la previsualización de mensajes de la lista de amigos',
        quickrep: 'Respuesta rápida',
        quickrepdesc: 'Oculta la opción de respuesta rápida del chat',
        offlinefriend: 'Amigos desconectados',
        offlinefrienddesc: 'Oculta los amigos desconectados de la lista de amigos',
        invisiblefriend: 'Amigos invisibles',
        invisiblefrienddesc: 'Oculta los amigos con un estado en línea desconocido de la lista de amigos',
        notification: 'Notificaciones',
        notificationdesc: 'Muestra una notificación cuando Stadia Enhanced se actualiza a una nueva versión ("Automático" se oculta después de 5 segundos, "Manual" permanece hasta la interacción del usuario)',
        streammode: 'Modo retransmisión',
        streammodedesc: 'Permite ocultar ciertos elementos (por ejemplo: la lista de amigos) mientras retransmites (a través de un programa externo como OBS o Discord)',
        forcefocus: 'Forzar foco',
        forcefocusdesc: 'Habilita para evitar que los juegos se pausen automáticamente cuando no esté enfocado',
        catprev: 'Previsualización de categorías',
        catprevdesc: 'Oculta las etiquetas de categoría cuando el cursor pasa sobre un juego de la página de inicio',
        streammondesc: 'Activa automáticamente el monitor de retransmisión al ejecutar un juego',
        resolutiondesc: 'Máxima resolución a la que pueden alcanzar los juegos. Para resoluciones 1440p (Quad HD) o 2160p (4K UHD) es necesario el códec VP9',
        codecdesc: 'Códec usado por los juegos',
        confirmreset: '¿Estás seguro de querer restablecer los ajustes?',
        statistics: 'Estadísticas',
        gamesfinished: 'Juegos completados',
        achievements: 'Logros',
        achievementsunlocked: 'Logros desbloqueados',
        totalPlayTime: 'Tiempo total jugado',
        splitstore: 'Tienda a doble columna',
        splitstoredesc: 'Divide la lista de la tienda en dos columnas para una mayor legibilidad',
        inlineimage: 'Previsualización de imágenes',
        inlinedesc: 'Reemplaza los enlaces de imágenes para formatos de archivo comunes (jpg/gif/png) con una vista previa en la que se puede hacer clic',
        familyelements: 'Compartir con familia',
        familyelementsdesc: 'Oculta las opciones "Compartir este juego con la familia"',
        donations: 'Donaciones',
        reportbug: 'Reportar un problema',
        exportset: 'Exportar ajustes',
        importset: 'Importar ajustes',
        importerror: 'El archivo que estás intentando abrir no contiene un perfil de Stadia Enhanced',
        resetsettings: 'Restablecer ajustes'
    },
        
[translations.zip](https://github.com/ChristopherKlay/StadiaEnhanced/files/9185491/translations.zip)

**Reply by ChristopherKlay** [reply]
Will be part of `1.20.8` going live in just a moment.

Thank you once again! (:

**Reply by rdrenedo** [reply]
New update with translate "progames" and minor corrections:

        es: { // https://github.com/ChristopherKlay/StadiaEnhanced/discussions/67
        default: 'Por defecto',
        native: 'Nativa',
        hide: 'Ocultar',
        show: 'Mostrar',
        total: 'Total',
        visible: 'Visible',
        hidden: 'Oculto',
        enabled: 'Activado',
        disabled: 'Desactivado',
        auto: 'Automático',
        manual: 'Manual',
        all: 'Todo',
        locked: 'Bloqueado',
        complete: 'Completo',
        incomplete: 'Incompleto',
        games: 'Juegos',
        allgames: 'Todos los juegos',
        leavepro: 'Dejarán Stadia Pro',
        bundles: 'Paquetes',
        addons: 'Complementos',
        wishlist: 'Lista de deseos',
        responsive: 'Adaptativo',
        windowed: 'Modo ventana',
        fullscreen: 'Pantalla completa',
        progames: 'Juegos de Stadia PRO',
        onsale: 'En oferta',
        prodeals: 'En oferta Stadia Pro',
        trials: 'Versiones de prueba',
        userprofile: 'Perfil',
        usermedia: 'Capturas y estados del juego',
        searchbtnbase: 'Buscar en',
        avatarpopup: 'URL del nuevo avatar (vacío por defecto):',
        date: 'Fecha',
        time: 'Hora',
        sessiontime: 'Duración de la sesión',
        codec: 'Códec',
        resolution: 'Resolución',
        hardware: 'Hardware',
        software: 'Software',
        trafficsession: 'Tráfico de la sesión',
        trafficcurrent: 'Tráfico actual',
        trafficaverage: 'Tráfico promedio',
        packetloss: 'Paquetes perdidos',
        framedrop: 'Imágenes perdidas',
        latency: 'Latencia',
        jitter: 'Jitter Buffer',
        decodetime: 'Tiempo de decodificación',
        compression: 'Compresión',
        detailed: 'Detallado',
        compact: 'Compacto',
        streammon: 'Monitor de retransmisión',
        stream: 'Retransmisión',
        network: 'Red',
        session: 'Sesión',
        extdetail: 'Detalles extendidos',
        maxresolution: 'Resolución máxima',
        fps: 'FPS',
        testdiscl: '<b>Aviso:</b> Este juego aún no se ha probado.',
        datadiscl: 'Esta es la velocidad máxima de fotogramas que se logra al jugar en modo 4K (requiere Stadia Pro). \
                    En los juegos con interruptor "gráficos/rendimiento", se eligió gráficos. \
                    Estos datos son proporcionados por <a href="https://twitter.com/OriginaIPenguin" target="_blank">@OriginaIPenguin</a> \
                    y se puede encontrar la base de datos completa <a href="https://linktr.ee/StadiaDatabase" target="_blank">aquí</a>.',
        noteOne: 'Modo 4K',
        noteTwo: 'Interruptor gráficos/rendimiento',
        noteThree: '60 FPS en modo 1080p',
        noteFour: '30 FPS en modo 1080p',
        noteFive: 'No es compatible con modo 4K',
        unsupported: 'No es compatible',
        crossfriends: 'Sin sistema Crossplay',
        filtersettings: 'Ajustes de filtro',
        saturation: 'Saturación',
        contrast: 'Contraste',
        brightness: 'Brillo',
        sharpen: 'Nitidez',
        community: 'Comunidad Stadia',
        speedtest: 'Test de velocidad',
        quickaccess: 'Acceso rápido',
        messages: 'Mensajes',
        comfeature: 'Comunidad',
        avatar: 'Avatar',
        interface: 'Interfaz',
        shortcut: 'Icono Stadia',
        shortcuttitle: 'Instala un acceso directo de',
        shortcutdesc: 'Permite instalar un acceso directo de un juego',
        stadiadatabase: 'Detalles extendidos',
        stadiadatabasedesc: 'Muestra una sección de "Detalles extendidos" en la página de la tienda de juegos, que muestra la velocidad de fotogramas, la resolución y más sobre el juego',
        contactdev: 'Contactar con el desarrollador',
        stadiahunters: 'Stadia Hunters',
        stadiahuntersdesc: 'Acceso a la comunidad de Stadia Hunters, que incluye seguimiento de logros, guías, tablas de clasificación y más. El compañero perfecto para los cazadores de logros',
        stadiahunterstitle: 'en Stadia Hunters',
        stadiahunterslogin: 'Iniciar sesión',
        stadiahuntersnotfound: 'Usuario no encontrado',
        stadiahunterslevel: 'Nivel',
        stadiahuntersworldrank: 'Rango mundial',
        stadiahuntersxphover: 'Progreso de nivel',
        gridsize: 'Tamaño de la cuadrícula',
        griddesc: 'Cambia la cantidad de juegos por fila en la biblioteca',
        clock: 'Reloj',
        clockdesc: 'Muestra la hora actual en la lista de amigos, como superposición dentro del juego o ambas cosas',
        friendslist: 'Lista de amigos',
        igoverlay: 'Superposición dentro del juego',
        listoverlay: 'Lista y superposición',
        filter: 'Filtro de juegos',
        filterdesc: 'Te permite ordenar tu biblioteca escondiendo juegos. El filtro se puede alternar a través del símbolo, arriba a la derecha, arriba de tus juegos en la biblioteca',
        invitebase: 'Copiar el enlace de invitación',
        inviteactive: 'Enlace del perfil copiado',
        gamelabel: 'Etiqueta Pro',
        gamelabeldesc: 'Elimina etiquetas "Pro" de los juegos en la pantalla de inicio',
        dimoverlay: 'Superposición atenuada',
        dimoverlaydesc: 'Elimina el efecto de atenuación al abrir el menú de Stadia durante el juego',
        homegallery: 'Galería de capturas',
        homegallerydesc: 'Oculta el área de "Capturas" de la parte inferior de la pantalla de inicio',
        quickprev: 'Previsualización de mensajes',
        quickprevdesc: 'Oculta la previsualización de mensajes de la lista de amigos',
        quickrep: 'Respuesta rápida',
        quickrepdesc: 'Oculta la opción de respuesta rápida del chat',
        offlinefriend: 'Amigos desconectados',
        offlinefrienddesc: 'Oculta los amigos desconectados de la lista de amigos',
        invisiblefriend: 'Amigos invisibles',
        invisiblefrienddesc: 'Oculta los amigos con un estado en línea desconocido de la lista de amigos',
        notification: 'Notificaciones',
        notificationdesc: 'Muestra una notificación cuando Stadia Enhanced se actualiza a una nueva versión ("Automático" se oculta después de 5 segundos, "Manual" permanece hasta la interacción del usuario)',
        streammode: 'Modo retransmisión',
        streammodedesc: 'Permite ocultar ciertos elementos (por ejemplo: la lista de amigos) mientras retransmites (a través de un programa externo como OBS o Discord)',
        forcefocus: 'Fuera de foco',
        forcefocusdesc: 'Habilita para evitar que el juego se pause automáticamente cuando no está siendo capturado',
        catprev: 'Previsualización de categorías',
        catprevdesc: 'Oculta las etiquetas de categoría cuando el cursor pasa sobre un juego de la página de inicio',
        streammondesc: 'Activa automáticamente el monitor de retransmisión al ejecutar un juego',
        resolutiondesc: 'Máxima resolución que pueden alcanzar los juegos. Para resoluciones 1440p o 2160p es necesario el códec VP9',
        codecdesc: 'Códec utilizado por los juegos',
        confirmreset: '¿Estás seguro de querer restablecer los ajustes?',
        statistics: 'Estadísticas',
        gamesfinished: 'Juegos completados',
        achievements: 'Logros',
        achievementsunlocked: 'Logros desbloqueados',
        totalPlayTime: 'Tiempo total jugado',
        splitstore: 'Tienda a doble columna',
        splitstoredesc: 'Divide la lista de la tienda en dos columnas',
        inlineimage: 'Previsualización de imágenes',
        inlinedesc: 'Reemplaza los enlaces de imágenes para formatos de archivo comunes (jpg/gif/png) con una vista previa en la que se puede hacer clic',
        familyelements: 'Compartir con familia',
        familyelementsdesc: 'Oculta las opciones "Compartir este juego con la familia"',
        donations: 'Donaciones',
        reportbug: 'Reportar un problema',
        exportset: 'Exportar ajustes',
        importset: 'Importar ajustes',
        importerror: 'El archivo que estás intentando abrir no contiene un perfil de Stadia Enhanced',
        resetsettings: 'Restablecer ajustes'
    },

[translations.zip](https://github.com/ChristopherKlay/StadiaEnhanced/files/9215005/translations.zip)

Thanks again!!

**Reply by ChristopherKlay** [reply]
Will be in the next patch (:
